site/src/modules/dashboard/Navbar/DeploymentDropdown.tsx

```diff
@@ -109,16 +109,7 @@ const DeploymentDropdownContent: FC<DeploymentDropdownProps> = ({
 					onClick={onPopoverClose}
 				>
 					Organizations
-				</MenuItem>
-			)}
-			{canViewAllUsers && (
```

Comment:
I worry that this is going to be one of those things that customers with lots of screenshots will really dislike. Can we loop @bpmct in on this so we can communicate this change clearly and hopefully mitigate frustrations?

Reply:
Ideally, the figma designs would already have been reviewed to validate this type of change. Almost everything about this PR would change existing screenshots. We should converge on a process after the holidays so we don't have to worry about these types of changes during PR review.

Reply:
yeah but changing things stylistically is very different from removing a path that users are used to and have documented. I doubt customers care nearly as much when things look slightly different but have the same structure as they do when we remove the button they put a screenshot of in their documentation entirely.